src/machine/machinectl.c

```diff
@@ -2731,6 +2732,7 @@ static int machinectl_main(int argc, char *argv[], sd_bus *bus) {
                 { "clone",           3,        3,        0,            clone_image       },
                 { "read-only",       2,        3,        0,            read_only_image   },
                 { "start",           2,        VERB_ANY, 0,            start_machine     },
+                { "stop",            2,        VERB_ANY, 0,            poweroff_machine  },
```

Comment:
This should really be placed next to the "poweroff" line I think, and a comment at the end `/* Convenience alias */` or so would be great.